lib/analytics_events_documenter.rb

```diff
@@ -156,9 +156,12 @@ def as_json
 
   private
 
-  # Naive attempt to pull tracked event string from source code
+  # Naive attempt to pull tracked event string or symbol from source code
   def extract_event_name(method_object)
-    m = /track_event\(\s*[:"'](?<event_name>[^"']+)["',)]/.match(method_object.source)
```

Comment:
Was the problem more that the negated `[^"']+` didn't include the comma from the following character set `["',)]`?

i.e. 

```diff
- m = /track_event\(\s*[:"'](?<event_name>[^"']+)["',)]/.match(method_object.source)
+ m = /track_event\(\s*[:"'](?<event_name>[^"',]+)["',)]/.match(method_object.source)
```

Alternatively we could change that capture from greedy to lazy:

```diff
- m = /track_event\(\s*[:"'](?<event_name>[^"']+)["',)]/.match(method_object.source)
+ m = /track_event\(\s*[:"'](?<event_name>.+?)["',)]/.match(method_object.source)
```

Reply:
Technically a comma would be valid inside the string of an event name, so I didn't want to try that. But realistically that would work